server/index.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const webpack = require('webpack');
+const webpackDevMiddleware = require('webpack-dev-middleware');
```

Comment:
This shouldn't be required in all cases. Needs to be only required in `development` otherwise middleware will become a prod dep which isn't good. Prod will use an actual bundle your code needs to accommodate both

Reply:
Since we want to only use webpack dev middleware and webpack hot middleware only in development, would I need to create separate server files for development and production?